Randolph-dev/Druid_Partnering_Project_React24k
# Scrum 134 paragraph content type

## Related ticket:
[SCRUM-134](https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-134?atlOrigin=eyJpIjoiYzQ1ZDMwZjIzODViNDhjY2IxMjY3YWM4N2E0MGI4NGEiLCJwIjoiaiJ9)

## In this PR:
- Update db with the last section of the front-page
- Finish the frontpage rendering
- Modify the Redux state to store page data
- Some small refactoring and export Drupal config

## Testing instruction
- Checkout branch
- Import db and config
- Check the Frontpage

[SCRUM-134]: https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-134?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
